builder/googlecompute/artifact.go

```diff
@@ -43,8 +45,14 @@ func (a *Artifact) String() string {
 }
 
 func (a *Artifact) State(name string) interface{} {
-	if _, ok := a.StateData[name]; ok {
-		return a.StateData[name]
+	if name == registryimage.ArtifactStateURI {
```

Comment:
There's some rich metadata that we can extract from the Artifact config here if set. I will revisit this tomorrow.

Reply:
Sounds good.

Reply:
Labels added. Thanks again for the review.